test/task_test.rb

```diff
@@ -54,14 +54,14 @@ def mock_all_sensor_registers
 
     def get_expected_sample
         sample = Types.water_probe_acquanativa_ap3.ProbeMeasurements.new
-        sample.oxygen_concentration = @probe_measurements_raw[0] * 1e-8
+        sample.oxygen_concentration = @probe_measurements_raw[0] / 100.0 * 1e-6 / 1e-3
         sample.oxygen_saturation = @probe_measurements_raw[1] * 1e-4
-        sample.temperature.kelvin = 275.15;
-        sample.pH = @probe_measurements_raw[3] * 1e-2;
-        sample.conductivity = @probe_measurements_raw[4] * 1e-10;
-        sample.salinity = @probe_measurements_raw[5] * 1e-2
-        sample.dissolved_solids = @probe_measurements_raw[6] * 1e-8
-        sample.specific_gravity = @probe_measurements_raw[7] * 1e-2
+        sample.temperature.kelvin = 275.15
+        sample.pH = @probe_measurements_raw[3] / 100.0
+        sample.conductivity = @probe_measurements_raw[4] / 100.0 * 1e-6 / 1e-2
+        sample.salinity = @probe_measurements_raw[5] / 100.0 * 1e-3
+        sample.dissolved_solids = @probe_measurements_raw[6] / 100.0 * 1e-6
+        sample.specific_gravity = @probe_measurements_raw[7] / 100.0
         sample.oxidation_reduction_potential = @probe_measurements_raw[8] * 1e-3
         sample.turbidity = @probe_measurements_raw[9]
         sample.height = @probe_measurements_raw[10]
```

Comment:
This can be removed from this PR. This test was fixed here: https://github.com/tidewise/drivers-water_probe_acquanativa_ap3/pull/4